backend/src/database/state-migrations/migrations.ts

```diff
@@ -52,55 +54,59 @@ export const migrations: {
     8: treatmentSystemImprovements8,
 };
 
-export async function migrateInDatabaseTo(
-    targetStateVersion: number,
-    currentStateVersion: number,
+/**
+ *
+ * @param targetStateVersion The new state version the exercise should have.
+ *          `'latest'` to always migrate as far as possible.
+ *          Must be at least as big as the current sate version.
+ */
+export async function migrateInDatabase(
+    targetStateVersion: number | 'latest',
```

Comment:
I think we could just omit this parameter and always migrate to the latest version. YAGNI

The same for the other migrate function.